libs/design-system/layouts/table-of-contents/src/lib/table-of-contents-layout.component.scss

```diff
@@ -31,4 +31,10 @@
       margin: 0 0.75rem 1rem;
     }
   }
+
+  ::ng-deep markdown {
```

Comment:
This does not belong in the layout component. Should be added in the `styles/` directory instead